docs/visual-basic/getting-started/index.md

```diff
@@ -27,32 +27,33 @@ Lists changes in this release that might prevent an application from compiling o
 Provides a list of Web sites and newsgroups that can help you find answers to common problems.  
 
 ## See also
+
 - [Get Visual Basic](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)
- Provides download links for Visual Studio versions that include Visual Basic support, including free versions.  
+ Provides download links for Visual Studio versions that include Visual Basic support, including free versions.
```

Comment:
It would be best for the description to be list continuation text on a separate line from the link. Since the link and the description are on the same line, they're hard to parse:
![image](https://user-images.githubusercontent.com/10886961/51761178-3a26c480-2081-11e9-98a4-b1fe249dec15.png)

Reply:
Should I use double spaces or `<br />` to break the line?

Reply:
What I've started doing in these cases where is to have a \ at the end of the line like

```markdown
- [Get Visual Basic](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)\
Provides download links for Visual Studio versions that include Visual Basic support, including free versions.
```

Reply:
I never knew this syntax. What does it mean?

Reply:
It's somewhat new to me as well but it breaks the line as you can see how the same content renders:

- [Get Visual Basic](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)\
Provides download links for Visual Studio versions that include Visual Basic support, including free versions.

Reply:
Awesome suggestion @mairaw I didn't know about that either. Must nicer than the double spaces.

Reply:
I learned that with @mvelosop 😉

Reply:
Double line breaks are used in the corresponding [C# article](https://github.com/dotnet/docs/blob/master/docs/csharp/getting-started/index.md).